### PR TITLE
Victor K api-registration (1st submission)

### PR DESCRIPTION
**Pull request by vkalinkin**
api registration

### CODE REVIEW

File: api-registration/server/index.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  }
+
+  argon2.hash(password)
+    .then(result => {
+      const params = [username, result];
+      const sql = `
+  insert into "users"
+  ("username", "hashedPassword")
+  values ($1, $2)
+  returning *
```
Comment (scbowler) [inline api-registration/server/index.js:34]: You don't want to return all the data to the client, especially the hashedPassword, easiest way to control that is to control what is returned from the SQL statement.

```SQL
returning "userId", "username", "createdAt"
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good but 1 small change needed. Let em know if you have any questions.

**Comment by vkalinkin** [comment]
api registration 2

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**